boards/arm/stm32_min_dev/doc/stm32_min_dev.rst

```diff
@@ -0,0 +1,114 @@
+.. _stm32_min_dev:
+
+STM32 Minimum Development Board
+###############################
+
+Overview
+********
+
+The STM32 Minimum Development Board, is a popular and inexpensive
+breadboard-friendly breakout board for the `STM32F103x8`_ CPU. Zephyr
+applications use the stm32_min_dev board configuration to run on these boards.
+
+.. figure:: img/stm32_min_dev.jpg
+     :width: 500px
+     :align: center
+     :height: 350px
+     :alt: STM32 Minimum Development Board
+
+     STM32 Minimum Development Board
+
+As the name suggests, these boards have the bare minimum components required to
+power on the CPU. For practical use, you'll need to add additional components
+and circuits using a breadboard, for example.
+
+Pin Mapping
+===========
+
+This port is a starting point for your own customizations and not a complete
+port for a specific board. Most of the GPIOs on the STM32 SoC has been exposed
+in the external header with silk screen labels that match the SoC's pin names.
+
+Each board vendor has their own variations in pin mapping on their boards'
+external connectors and placement of components. Many vendors use port PB12 for
+connecting an LED, so only this device is supported by our Zephyr port.
+Additional device support is left for the user to implement.
+
+More information on hooking up peripherals and lengthy how to articles can be
+found at `EmbedJoural`_.
```

Comment:
PR https://github.com/zephyrproject-rtos/zephyr/pull/565 (on the arm branch) fixes issues with this .rst file